Examples/Run/Geant4/CMakeLists.txt

```diff
@@ -1,17 +1,18 @@
 add_executable(
-  ActsSimGeantinoRecording
-  RunSimGeantinoRecording.cpp)
+  ActsExampleGdmlGeantinoRecording
+  GdmlGeantinoRecording.cpp)
 target_link_libraries(
-  ActsSimGeantinoRecording
+  ActsExampleGdmlGeantinoRecording
   PRIVATE
     ActsExamplesCommon
     ActsExamplesDetectorDD4hep
     ActsExamplesFramework
     ActsExamplesGeant4
-    ActsExamplesGeant4DD4hep
     ActsExamplesIoRoot
     Boost::program_options)
 
 install(
-  TARGETS ActsSimGeantinoRecording
+  TARGETS ActsExampleGdmlGeantinoRecording
```

Comment:
I would suggest to stay with the `ActsSim...` prefix. The `ActsExample...` prefix for executable is only used if the tools does not fit in either of the `ActsGen...`, `ActsSim...`, `ActsRec...` categories.

I would also suggest to invert the naming hierarchy, i.e. use
```
ActsSimGeantinoRecordingGdml
ActsSimGeantinoRecordingDD4hep
```
as the executable names. I think it is more consistent and easier to use e.g. when you are using tab completion.

Reply:
Oh, I have practically finished my work on the 'ActsExamples` PR, which makes one single target naming for all examples:

`ActsExample<WhatItIs>` 

I removed the `Sim/Reco/Gen` distinction, because soon or later it will give us problems.

Reply:
Ok, lets stay with `ActsExample...`. I liked the `Sim/Reco/Gen` prefixes but I can see your point. Also, I am usually the one calling for consistency above all else :grinning:.

Reply:
Ok done.